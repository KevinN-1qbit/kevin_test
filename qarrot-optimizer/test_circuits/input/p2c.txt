Rotate 2: ZXI
Rotate -2: ZII
Rotate -2: IXI
Rotate 2: IZI
Rotate 2: IXI
Rotate 2: IZI
Rotate 1: IZI
Rotate 2: IZX
Rotate -2: IZI
Rotate -2: IIX
Rotate 2: ZXI
Rotate -2: ZII
Rotate -2: IXI
Rotate 2: XIZ
Rotate -2: IIZ
Rotate -2: XII
Rotate 2: IZX
Rotate -2: IZI
Rotate -2: IIX