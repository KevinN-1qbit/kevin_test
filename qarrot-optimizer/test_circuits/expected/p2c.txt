Rotate 1: IXI
Measure +: ZXZ
Measure +: ZXI
Measure +: ZIZ
